Stripe/PublicHeaders/STPPaymentConfiguration.h

```diff
@@ -14,6 +14,21 @@
 
 NS_ASSUME_NONNULL_BEGIN
 
+
+/**
```

Comment:
I think we should go into a bit more detail here, and frame it from the perspective of the user's experience.

> Options for 3D Secure support.
> 
> STPThreeDSecureSupportTypeDisabled: Your customer will never be prompted for 3DS verification.
> 
> STPThreeDSecureSupportTypeStatic: Your customer will be prompted for 3DS verification if available. If 3DS is not available, the original card source will be passed to your app. Note that if you charge a card source, you will not be protected from fraud by 3DS. If you want to reject all card payments that haven't passed 3DS verification, you should update your logic accordingly in `paymentContext:didCreatePaymentResult:completion:`: if the payment result is a card source, instead of passing the source to your backend to be charged, you should call the completion block with an error.
>